### PR TITLE
DOC: Fixed PR08 docstring errors in pandas.DataFrame

### PR DESCRIPTION
**Pull request by jb8429**
This relates to: [27977](https://github.com/pandas-dev/pandas/issues/27977). I have fixed the doc PR08 formatting issues for:
```
pandas.DataFrame.insert: Parameter "column" description should start with a capital letter
pandas.DataFrame.apply: Parameter "raw" description should start with a capital letter
pandas.DataFrame.pipe: Parameter "func" description should start with a capital letter
pandas.DataFrame.pipe: Parameter "args" description should start with a capital letter
pandas.DataFrame.pipe: Parameter "kwargs" description should start with a capital letter
pandas.DataFrame.corr: Parameter "method" description should start with a capital letter
pandas.DataFrame.corrwith: Parameter "axis" description should start with a capital letter
pandas.DataFrame.corrwith: Parameter "method" description should start with a capital letter
pandas.DataFrame.drop_duplicates: Parameter "keep" description should start with a capital letter
pandas.DataFrame.duplicated: Parameter "keep" description should start with a capital letter
pandas.DataFrame.idxmax: Parameter "axis" description should start with a capital letter
pandas.DataFrame.idxmin: Parameter "axis" description should start with a capital letter
pandas.DataFrame.unstack: Parameter "fill_value" description should start with a capital letter
pandas.DataFrame.plot: Parameter "kind" description should start with a capital letter
pandas.DataFrame.plot: Parameter "style" description should start with a capital letter
pandas.DataFrame.to_feather: Parameter "fname" description should start with a capital letter
pandas.DataFrame.to_msgpack: Parameter "path" description should start with a capital letter
pandas.DataFrame.to_msgpack: Parameter "append" description should start with a capital letter
pandas.DataFrame.to_msgpack: Parameter "compress" description should start with a capital letter
pandas.DataFrame.to_clipboard: Parameter "excel" description should start with a capital letter
```

- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

Will continue to work through all PR08 docstring errors.

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -6233,8 +6235,8 @@ def unstack(self, level=-1, fill_value=None):
         ----------
         level : int, string, or list of these, default -1 (last level)
             Level(s) of index to unstack, can pass level name
-        fill_value : replace NaN with this value if the unstack produces
-            missing values
+        fill_value : sint, string or dict
```
Comment (gfyoung) [inline pandas/core/frame.py]: `sint` ?

**Comment by gfyoung** [comment]
So close!  If you can fix the merge conflict.

**Comment by jackbicknell14** [comment]
> So close! If you can fix the merge conflict.

Thanks for the help @gfyoung 

Have updated and resolved the merge conflict. Could you also check out my other PR for similar doc issues in PR[28571](https://github.com/pandas-dev/pandas/pull/28571)

Thanks!

**Review by gfyoung — APPROVED** [review]

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @jackbicknell14 !

**Pull request merged**